src/lab-tabs/data-table-extensions/pending-review-lab-request-table.extension.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import OrdersDataTable from '../../components/orders-table/orders-data-table.component';
+
+const PendingReviewLabRequestsTable: React.FC = () => {
+  return (
+    <OrdersDataTable
+      excludeColumns={[]}
+      fulfillerStatus="ON_HOLD"
```

Comment:
I don't think that `ON_HOLD` is the right value to use here. We've been drawing from versions of the [FHIR status](https://hl7.org/fhir/R4/valueset-medicationrequest-status.html) valueset for this. Orders that are not yet finalized and need to be approved should be marked as being in `DRAFT` status. `ON_HOLD` indicates that no processing is happening.